addons/survey/controllers/main.py

```diff
@@ -25,11 +25,15 @@ class Survey(http.Controller):
     # ACCESS
     # ------------------------------------------------------------
 
-    def _fetch_from_access_token(self, survey_token, answer_token):
+    def _fetch_from_access_token(self, survey_token, answer_token, survey_id=False):
```

Comment:
Well, would probably renamed `fetch_answer` or something since we're removing the "from_access_token" part